__tests__/api.test.js

```diff
@@ -27,17 +26,78 @@ describe('API testing', () => {
                 })
         })  
     });
-        test('404 - It should return an error when the wrong path is provided', () => {
+        test('404, It should return an error when the wrong path is provided', () => {
             return request(app)
             .get('/api/topical')
             .expect(404)
             .then(({body : {msg}}) => {
              expect(msg).toBe("404 Not Found")   
             })
         });
-        // test('GET /api/description', () => {
-
-        //     expect().toEqual();
-        // });
     })
+    describe('GET /api/articles', () => {
+        test('should return array of article objects including a comment_count property ', () => {
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body})=> {
+            const {articles} = body
+                expect(articles).toHaveLength(12)
+                articles.forEach((article) => {
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
+                        topic: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(Number)
+                    })
+                })
+            })
+        })
+        test("200 It should should return article objects in date descending order", () => {
+            return request(app)
+              .get("/api/articles")
+              .expect(200)
+              .then(({ body: { articles } }) => {
+                expect(articles).toBeSortedBy("created_at", {descending: true });
+              });
+            });
+    })
+    describe("GET /api/articles/:article_id", () => {
+        test("should return an array of an article objects that matches the passed article_id", () => {
+          const articleObject = {
+          author: "rogersop",
```

Comment:
The one key you havent checked is the most important, you need to check it has the id you've requested

Reply:
udpated test